pkg/cmd/service/service.go

```diff
@@ -145,12 +150,12 @@ type Config struct {
 // DefaultConfig returns the OpenFGA server default configurations.
 func DefaultConfig() *Config {
 	return &Config{
-		OpenFGA: OpenFGAConfig{
-			MaxTuplesPerWrite:             100,
-			MaxTypesPerAuthorizationModel: 100,
-			ChangelogHorizonOffset:        0,
-			ResolveNodeLimit:              25,
-		},
+		MaxTuplesPerWrite:             100,
+		MaxTypesPerAuthorizationModel: 100,
+		ChangelogHorizonOffset:        0,
+		ResolveNodeLimit:              25,
+		ListObjectsDeadline:           3 * time.Second, // we have a 3-second timeout anyway
```

Comment:
Where do we have that timeout? 🤔

Reply:
https://github.com/openfga/openfga/blob/a5fb26c2ac203dc8c3c9f89db18cad3205e89a4b/pkg/cmd/service/service.go#L172